### DISCUSSION TITLE
Foundry Installation Error

### DISCUSSION THREAD

**Discussion by Suzie1899** [Cyfrin/foundry-full-course-cu#1107] [Q&A]
i tried installing foundry but I'm getting this error: sumedha@Sumedhas-MacBook-Air ~ % curl -Lk https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  1942  100  1942    0     0   2923      0 --:--:-- --:--:-- --:--:--  2923
Installing foundryup...
curl: (35) Recv failure: Connection reset by peer

**Comment by 0xusmanf** [comment]
Have you tried installing it with a different network? This error indicates it is a network connection error, you can find more about it [here](https://www.phind.com/search?cache=vsrryknt1hgod39u6ej75kre).

**Reply by Suzie1899** [reply]
Thank you that worked!

**Reply by Kanishka227** [reply]
I am facing the same issue .. How did you manage to resolve?

**Reply by PiyushFx** [reply]
still i am getting error
